feature-libs/product/image-zoom/components/product-image-zoom/product-image-zoom-product-images/product-image-zoom-product-images.component.ts

```diff
@@ -18,6 +20,10 @@ export class ProductImageZoomProductImagesComponent extends ProductImagesCompone
     super(currentProductService);
   }
 
+  getProduct(): Observable<Product> {
```

Comment:
We don't need this method. `product$` is already exposed thanks to extending `ProductImagesComponent`. We just need to make it public for the template like `product$: Observable<Product> = this.product$;`.

Reply:
Done